agent/lib/console_server.js

```diff
@@ -51,17 +52,12 @@ function ConsoleServer (address_ctrl) {
     this.amqp_container.on('sender_open', function (context) {
         self.subscribe(context.connection.remote.open.container_id, context.sender);
     });
-    this.amqp_container.on('sender_close', function (context) {
+    function unsubscribe (context) {
         self.unsubscribe(context.connection.remote.open.container_id);
```

Comment:
The condition if (context.connection.remote.open)  is missing from the disconnected case, is it ok?